### PR TITLE
Kb for auxiliary/scanner/ftp/ftp_version

### PR DESCRIPTION
**Pull request by itsmeroy2012**
### Docs for auxiliary/scanner/ftp/ftp_version

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/ftp/ftp_version.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+1.  ```apt-get install vsftpd```
+2. Allow local users to log in and to allow ftp uploads by editing file /etc/vsftpd.conf uncommenting the following:
+	```
+	local_enable=YES
+	write_enable=YES
+	chroot_list_enable=YES
+	chroot_list_file=/etc/vsftpd.chroot_list
+	```
+3. **IMPORTANT:** For allowing anonymous access set ```anonymous_enable=YES```
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/ftp_version.md]: can be removed

File: documentation/modules/auxiliary/scanner/ftp/ftp_version.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/ftp/ftp_version```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set RPORT [IP]```
+4. Do: ```run```
+
+## Sample Output
+```
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/ftp_version.md]: ### on vsFTPd 3.0.3 on Kali

File: documentation/modules/auxiliary/scanner/ftp/ftp_version.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+msf auxiliary(ftp_version) > exploit
+
+[*] 127.0.0.1:21          - FTP Banner: '220 (vsFTPd 3.0.3)\x0d\x0a'
+[*] Scanned 1 of 1 hosts (100% complete)
+[*] Auxiliary module execution completed
+msf auxiliary(ftp_version) > 
+```
+## Confirming using NMAP
+```
+root@kali:~# nmap -sV 127.0.0.1
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/ftp_version.md]: do it again w/ -p21 to save the other output

File: documentation/modules/auxiliary/scanner/ftp/ftp_version.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+Starting Nmap 7.40SVN ( https://nmap.org ) at 2017-04-24 00:12 IST
+Nmap scan report for localhost (127.0.0.1)
+Host is up (0.0000020s latency).
+Not shown: 997 closed ports
+PORT     STATE SERVICE    VERSION
+21/tcp   open  ftp        vsftpd 3.0.3
+111/tcp  open  rpcbind    2-4 (RPC #100000)
+5432/tcp open  postgresql PostgreSQL DB 9.5.4
+Service Info: OS: Unix
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/ftp_version.md]: no need for the rest of the output after this line

**Comment by itsmeroy2012** [comment]
Made the changes @h00die

**Comment by h00die** [comment]
one small change https://github.com/rapid7/metasploit-framework/commit/4c71a9908d72007e8c2a64eeb4c98bf9cae37e4a
looked right in github, but not kali an msf.

**Comment by h00die** [comment]
# Release Notes
Adds docs for `ftp_version`

**Comment by h00die** [comment]
I think the issue may be that your editor (or you) used tabs instead of 4 space.  Some editors covert automatically.  Just something to watch for.

**Pull request merged**